UC1/src/main/java/es/uclm/esi/isoft2/a04/Persistence/BookingDAO.java

```diff
@@ -71,7 +67,7 @@ public void readBooking(Booking booking)
 
 	/**
 	 * @param booking The Booking instance to be created
-	 * @return The number of modified rows
+	 * @return The number of modified columns
 	 * @throws InstantiationException
```

Comment:
It is the number of rows, not colums. This mistake appears multiple times, and it should be fixed everywhere.